home/redux/SessionActions.ts

```diff
@@ -18,7 +18,12 @@ export default {
     return async dispatch => {
       const session = await LocalStorage.getSessionAsync();
       if (session) {
-        await AuthApi.signOutAsync(session.sessionSecret);
+        try {
+          await AuthApi.signOutAsync(session.sessionSecret);
+        } catch (e) {
+          // continue to clear out session in redux and local storage even if API logout fails
+          console.log({ e });
```

Comment:
console.error, provide some more context so if we see this error we know what it's about:

```suggestion
          console.error('Something went wrong when signing out:', e);
```